### ISSUE TITLE
reasons to differentiate between a direct attribute Mountable and the property set Pset_KerbCommon

### ISSUE THREAD

**Issue by TLiebich** [bSI-InfraRoom/IFC-Specification#208]
why is the attribute Mountable be directly associated to IfcKerb, and not added to Pset_KerbCommon. Direct attributes to subtypes of IfcProduct are (mostly?) deprecated in IFC4 and it is not advisible to add new ones. It is a better practise to use Psets consistently.

At the same time, the attribute PredefinedType :: IfcKerbTypeEnum is missing. It is part of the IFC modeling style, that all leafnode subtypes of IfcElement do have a predefined type (even if there would only be USERDEFINED; NOTDEFINED as a placehollder for now.

There is a Pset_KerbStone - does it suggest that STONE should be a predefined type of IfcKerb?

**Comment by SergejMuhic** [comment]
Good point! This somehow got lost before the process was established but I remember it being agreed to be implemented as you propose with the addition of MOUNTABLE and NONMOUNTABLE/NOTMOUNTABLE to the IfcKerbTypeEnum.
